src/main/java/com/limechain/network/NetworkService.java

```diff
@@ -47,12 +51,11 @@
  */
 @Log
 @Getter
-@Component
-public class Network {
+@Service
+public class NetworkService implements NodeService {
```

Comment:
Curious that you decided to change the `@Component` annotation into a `@Service` one. I think all of our services are annotated with `@Component` so there's probably no reason to break that trend. There is no functional difference between the annotations.